.storybook/styles.scss

```diff
@@ -1,20 +1,6 @@
 @import '../src/styles/libraries/libraries';
 @import '../src/styles/index';
 
-html {
-  font-size: 10px;
-}
-
-body {
-  // overwrite mc-sass-lib default body black background
-  background-color: transparent !important;
-}
-
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
-  color: #000;
-}
+// Overrides masterclass default bg color
+// so that storybook can change bg
+body { background: transparent !important; }
```

Comment:
kill this too, we should move to not have a changeable background as it adds complexity without much benefit